starter-code/lib/office.js

```diff
@@ -0,0 +1,38 @@
+var _ = require('lodash');
+
+//1
+var opinions = [ "This is the best job ever",
+                 "Satisfied",
+                 "At least I get paid",
+                 "I'm looking for another job",
+                 "I don't want to answer"];
+
+//2
+function randomOpinion() {
+  var randomNumber =  _.random(0, opinions.length-1);
+  return opinions[randomNumber];
+}
+
+//3
+var newOpinions = [];
+
+function pushRandomOpinion(){
+  for (var i=0; i<10; i++){
+    newOpinions.push(randomOpinion());
+  }
+}
+
+pushRandomOpinion();
+//console.log(newOpinions);
+
+//4
+var fiveItemArray = [];
```

Comment:
Bravo!! habéis sacado el ejercicio!!! :D, ahora hay que dar un paso más ya que en el futuro vais a necesitar usar el "return". Os pongo el enlace a la learning para que le echéis un ojo
http://learn.ironhack.com/#/learning_unit/2095

Y os hago un ejemplo para que veáis el uso. Si no lo entendéis o tenéis dudas, preguntad!!! 
`function pushArray() {
   var fiveItemArray = []
   for (var i=0; i<5; i++){
     fiveItemArray.push(newOpinions);
   }
   return fiveItemArray
 }`

console.log(pushArray())